qds_sdk/cluster.py

```diff
@@ -383,27 +310,276 @@ def _parse_create_update(cls, args, action, api_version):
-                               of the cluster. Specified as JSON object (key-value pairs)
-                               e.g. --custom-ec2-tags '{"key1":"value1", "key2":"value2"}'
-                               """,)
+                               dest="node_bootstrap_file",
+                               help="""name of the node bootstrap file for this cluster. It
+                                    should be in stored in S3 at
+                                    <account-default-location>/scripts/hadoop/NODE_BOOTSTRAP_FILE
+                                    """, )
+
+        if api_version >= 1.3:
+            qubole_placement_policy_group = hadoop_group.add_mutually_exclusive_group()
+            qubole_placement_policy_group.add_argument("--use-qubole-placement-policy",
+                                                       dest="use_qubole_placement_policy",
+                                                       action="store_true",
+                                                       default=None,
+                                                       help="Use Qubole Block Placement policy" +
+                                                            " for clusters with spot nodes", )
+            qubole_placement_policy_group.add_argument("--no-use-qubole-placement-policy",
+                                                       dest="use_qubole_placement_policy",
+                                                       action="store_false",
+                                                       default=None,
+                                                       help="Do not use Qubole Block Placement policy" +
+                                                            " for clusters with spot nodes", )
+            fallback_to_ondemand_group = node_config_group.add_mutually_exclusive_group()
+            fallback_to_ondemand_group.add_argument("--fallback-to-ondemand",
+                                                    dest="fallback_to_ondemand",
+                                                    action="store_true",
+                                                    default=None,
+                                                    help="Fallback to on-demand nodes if spot nodes" +
+                                                         " could not be obtained. Valid only if slave_request_type is spot", )
+            fallback_to_ondemand_group.add_argument("--no-fallback-to-ondemand",
+                                                    dest="fallback_to_ondemand",
+                                                    action="store_false",
+                                                    default=None,
+                                                    help="Dont Fallback to on-demand nodes if spot nodes" +
+                                                         " could not be obtained. Valid only if slave_request_type is spot", )
+            if api_version == 1.3:
```

Comment:
Why are we checking the api version here since we are already inside `api_version == 1.3` (Line: 328)